packages/vaex-ml/vaex/ml/__init__.py

```diff
@@ -75,6 +75,17 @@ def wrapper(self, features=None, transform=False, **kwargs):
                             return dft
                         else:
                             return object
+
+                    # Add docstrings
+                    doc = '\n'
+                    for trait in class_spec['traits']:
+                        doc += f':param {trait["name"]}: {trait["help"]} \n'
+                    doc+= ':param transform: If True, return a shallow copy of the transformed DataFrame, otherwise the return fitted transformer. \n'
+                    try:
+                        wrapper.__doc__= class_spec['doc'] + doc
+                    except TypeError:
+                        wrapper.__doc__= class_spec['doc']
+
```

Comment:
Which TypeError? Only thing I can think of is that class_spec['doc'] is None, but then it should assign doc right?

Reply:
Ah you are right! Thanks!